egov-indexer/open-hearing-indexer.yml

```
@@ -0,0 +1,33 @@
+              - inJsonPath: $.caseTitle
+                outJsonPath: $.Data.hearingDetails.caseTitle
+              - inJsonPath: $.caseUuid
+                outJsonPath: $.Data.hearingDetails.caseUuid
+              - inJsonPath: $.hearingNumber
+                outJsonPath: $.Data.hearingDetails.hearingNumber
+              - inJsonPath: $.caseNumber
+                outJsonPath: $.Data.hearingDetails.caseNumber
+              - inJsonPath: $.stage
+                outJsonPath: $.Data.hearingDetails.stage
+              - inJsonPath: $.status
+                outJsonPath: $.Data.hearingDetails.status
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)